googlesearch/__init__.py

```diff
@@ -141,14 +143,13 @@ def get_tbs(from_date, to_date):
     :rtype: str
     :return: Dates encoded in tbs format.
     """
-    from_date = from_date.strftime('%m/%d/%Y')
-    to_date = to_date.strftime('%m/%d/%Y')
+    from_date, to_date = from_date.strftime('%m/%d/%Y'), to_date.strftime('%m/%d/%Y')
```

Comment:
Not sure what's the point of these kind of changes tbh

Reply:
Most proposed changes are either type-hinting or moving things inline.